src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -72,444 +104,413 @@ const Mentor = ({ dispatch, error, successPage }) => {
 
-                <Col
-                  span={14}
-                  offset={0}
-                  style={{ display: 'flex', justifyItems: 'left' }}
+            {/*Personal info*/}
+            <Row>
+              {/*First name*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="First name"
+                  type="text"
+                  name="first_name"
+                  rules={[
+                    {
+                      required: true,
+                      message: 'First name is required!',
+                    },
+                  ]}
+                  value={formValues.first_name}
+                  onChange={handleChange}
+                >
+                  <Input placeholder="First name" />
+                </Form.Item>
+              </Col>
+              {/*Last name*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="Last name"
+                  type="text"
+                  name="last_name"
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Last name is required!',
+                    },
+                  ]}
+                  value={formValues.last_name}
+                  onChange={handleChange}
+                >
+                  <Input placeholder="Last name" />
+                </Form.Item>
+              </Col>
+              {/*Email*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="Email"
+                  type="email"
+                  name="email"
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Email is required!',
+                    },
+                  ]}
+                  value={formValues.email}
+                  onChange={handleChange}
                 >
+                  <Input placeholder="Enter a valid email" />
+                </Form.Item>
+              </Col>
+              {/*Country*/}
+              <Col md={20} xs={24}>
+                <Form.Item
+                  label="Country"
+                  name="country"
+                  rules={[
+                    {
+                      required: true,
+                      message: 'Country is required!',
+                    },
+                  ]}
+                >
+                  <Select
+                    showSearch
+                    placeholder="- Select -"
+                    onChange={e => handleChange(e, 'select', 'country')}
+                  >
+                    {countries.map(country => (
+                      <Option key={country} value={country}>
+                        {' '}
+                        {country}{' '}
+                      </Option>
+                    ))}
+                  </Select>
+                </Form.Item>
+              </Col>
+              {/*State & State*/}
```

Comment:
My guess is that this was probably meant to be either "city and state" or just "state" with a separate "city" comment below.